docs/antrea-network-policy.md

```diff
@@ -689,6 +690,61 @@ The following kubectl commands can be used to retrieve CG resources:
     kubectl get cg.core.antrea.tanzu.vmware.com
 ```
 
+## Select Namespace by Name
+
+Kubernetes NetworkPolicies and Antrea-native NetworkPolicies allow selecting
+workloads from Namespaces with the use of a label selector (i.e. `namespaceSelector`).
+However, it is often desired that the Namespaces be selected directly by its `name`
+as opposed to using the `labels` associated with the Namespace. In order to select
+Namespaces by name, Antrea labels Namespaces with a reserved label `antrea.io/metadata.name`,
+which sets it's value to the name of the Namespace. Users can then use this label in the
```

Comment:
s/which sets it's value to the name of the Namespace/whose value is set to the Namespace's name

Reply:
done